core/frontend/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "vue": {
     "transpileDependencies": [
-      "vuetify"
+      "vuetify",
```

Comment:
This commit belongs to the beggining of the PR.

Reply:
done